Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml

```diff
@@ -1,6 +1,13 @@
+@inject Microsoft.Extensions.Configuration.IConfiguration config
 <!DOCTYPE html>
 <html lang="en" class="govuk-template">
 <head>
+    @{
+        var googleTagId = config["GoogleTagId"] ?? throw new Exception("Missing configuration key: GoogleTagId");
+    }
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=@googleTagId"></script>
+    <script>window.dataLayer = window.dataLayer || []; function gtag() { dataLayer.push(arguments); } gtag('js', new Date()); gtag('config', '@googleTagId');</script>
```

Comment:
Could we hide it if googleTagId is empty, rather than throw an exception?

Reply:
We can certainly hide it, though we will not know if the GTM is missing on website when config variable is not set.
Unless someone test is manually :)